### ISSUE TITLE
Allow usage of variables in v2

### ISSUE THREAD

**Issue by glumb** [bijoutrouvaille/fireward#21]
Security rules v2 allow for custom variable definitions:

```ts
function custom_is_UserReferences(data) { 
  let values = data.values();
  let size = data.size();
  return (data!=null && data is list && data.size() < 12)
          && size > 0 && is____UserReference(values[0])
          && size > 1 && is____UserReference(values[1])
          && size > 2 && is____UserReference(values[2])
          && size > 3 && is____UserReference(values[3])
          && size > 4 && is____UserReference(values[4])
          && size > 5 && is____UserReference(values[5])
          && size > 6 && is____UserReference(values[6])
          && size > 7 && is____UserReference(values[7])
          && size > 8 && is____UserReference(values[8])
          && size > 9 && is____UserReference(values[9])
          && size > 10 && is____UserReference(values[10]);
}
```

this fails with 
```terminal
function `custom_is_UserReferences` is missing a closing `}`
  on line 40, column 6
```

**Comment by bijoutrouvaille** [comment]
Thanks for the great info. The change to code should be easy enough. I'll plan on getting it done before the end of the week.

**Comment by bijoutrouvaille** [comment]
@glumb The custom let bindings feature should now be available in v 1.5.1. Thanks for reporting.

**Comment by glumb** [comment]
@bijoutrouvaille perfect, thanks for implementing! :)
